src/components/settings/Settings.h

```diff
@@ -184,6 +184,14 @@ namespace Pinetime {
         if (status != settings.notificationStatus) {
           settingsChanged = true;
         }
+	// Disable always on screen while sleep mode is enabled
+	if (settings.alwaysOnDisplay.enabled) {
+	  if (status == Notification::Sleep) {
+	    settings.alwaysOnDisplay.state = false;
+	  } else {
+            settings.alwaysOnDisplay.state = true;
+	  }
+	}
```

Comment:
i'm unsure if this code belongs here, or if we should check the notification status before sleeping instead